identifiers/country-ch/municipalities.csv

```diff
@@ -0,0 +1,2203 @@
+ocd-division/country:ch/canton:zh/municipality:0014,Wettswil am Albis
+ocd-division/country:ch/canton:zh/municipality:0021,Adlikon
+ocd-division/country:ch/canton:zh/municipality:0022,Benken (ZH)
+ocd-division/country:ch/canton:zh/municipality:0023,Berg am Irchel
+ocd-division/country:ch/canton:zh/municipality:0024,Buch am Irchel
+ocd-division/country:ch/canton:zh/municipality:0025,Dachsen
+ocd-division/country:ch/canton:zh/municipality:0026,Dorf
+ocd-division/country:ch/canton:zh/municipality:0027,Feuerthalen
+ocd-division/country:ch/canton:zh/municipality:0028,Flaach
+ocd-division/country:ch/canton:zh/municipality:0029,Flurlingen
+ocd-division/country:ch/canton:zh/municipality:0030,Andelfingen
+ocd-division/country:ch/canton:zh/municipality:0031,Henggart
+ocd-division/country:ch/canton:zh/municipality:0032,Humlikon
+ocd-division/country:ch/canton:zh/municipality:0033,Kleinandelfingen
+ocd-division/country:ch/canton:zh/municipality:0034,Laufen-Uhwiesen
+ocd-division/country:ch/canton:zh/municipality:0035,Marthalen
+ocd-division/country:ch/canton:zh/municipality:0037,Ossingen
+ocd-division/country:ch/canton:zh/municipality:0038,Rheinau
+ocd-division/country:ch/canton:zh/municipality:0039,Thalheim an der Thur
+ocd-division/country:ch/canton:zh/municipality:0040,Trüllikon
+ocd-division/country:ch/canton:zh/municipality:0041,Truttikon
+ocd-division/country:ch/canton:zh/municipality:0043,Volken
+ocd-division/country:ch/canton:zh/municipality:0051,Bachenbülach
+ocd-division/country:ch/canton:zh/municipality:0052,Bassersdorf
+ocd-division/country:ch/canton:zh/municipality:0053,Bülach
+ocd-division/country:ch/canton:zh/municipality:0054,Dietlikon
+ocd-division/country:ch/canton:zh/municipality:0055,Eglisau
+ocd-division/country:ch/canton:zh/municipality:0056,Embrach
+ocd-division/country:ch/canton:zh/municipality:0057,Freienstein-Teufen
+ocd-division/country:ch/canton:zh/municipality:0058,Glattfelden
+ocd-division/country:ch/canton:zh/municipality:0059,Hochfelden
+ocd-division/country:ch/canton:zh/municipality:0060,Höri
+ocd-division/country:ch/canton:zh/municipality:0061,Hüntwangen
+ocd-division/country:ch/canton:zh/municipality:0062,Kloten
+ocd-division/country:ch/canton:zh/municipality:0063,Lufingen
+ocd-division/country:ch/canton:zh/municipality:0064,Nürensdorf
+ocd-division/country:ch/canton:zh/municipality:0065,Oberembrach
+ocd-division/country:ch/canton:zh/municipality:0066,Opfikon
+ocd-division/country:ch/canton:zh/municipality:0067,Rafz
+ocd-division/country:ch/canton:zh/municipality:0068,Rorbas
+ocd-division/country:ch/canton:zh/municipality:0069,Wallisellen
+ocd-division/country:ch/canton:zh/municipality:0070,Wasterkingen
+ocd-division/country:ch/canton:zh/municipality:0071,Wil (ZH)
+ocd-division/country:ch/canton:zh/municipality:0072,Winkel
+ocd-division/country:ch/canton:zh/municipality:0081,Bachs
+ocd-division/country:ch/canton:zh/municipality:0082,Boppelsen
+ocd-division/country:ch/canton:zh/municipality:0083,Buchs (ZH)
```

Comment:
I assume the parts in parenthesis are to differentiate between municipalities of the same name in different cantons. Those shouldn't be necessary since the OCD ID uniquely identifies each municipality.

Reply:
The distinction would not be necessary, however I think they would still be useful as in some cases only the name will be shown to users (e.g. in a choice box).

Reply:
I think that the disambiguation strings in parentheses should be left out and we should stick with the names of the municipalities here. Whether or not those would be included in a UI would depend on that project's needs and I think it'd be easier to algorithmically add the disambiguation in than remove it.

Reply:
:+1: It is definitely easier to add them later on if needed.